client/state/themes/actions.js

```diff
@@ -765,3 +768,44 @@ export function confirmDelete( themeId, siteId ) {
 		);
 	};
 }
+
+function setPreviewingThemeData( themeId, siteId ) {
+	return {
+		type: PREVIEWING_THEME_DATA,
+		themeData: {
+			themeId,
+			siteId
```

Comment:
`siteId` isn't really theme data, so can we pass these things flat here (without the `themeData` wrapper)?